.github/workflows/update-syft-release.yml

```diff
@@ -0,0 +1,50 @@
+name: PR for latest Syft release
+on:
+  schedule:
+    - cron: "0 8 * * *" # 3 AM EST
+
+  workflow_dispatch:
+
+env:
+  GO_VERSION: "1.18.x"
+  GO_STABLE_VERSION: true
+
+jobs:
+  upgrade-syft:
+    runs-on: ubuntu-latest
+    if: github.repository == 'anchore/grype' # only run for main repo
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          stable: ${{ env.GO_STABLE_VERSION }}
+
+      - run: |
+          LATEST_VERSION=$(curl "https://api.github.com/repos/anchore/syft/releases/latest" 2>/dev/null | jq -r '.tag_name')
```

Comment:
Same comment added on the syft PR.

This is a really good change. Are there ways where we can make sure `LATEST_VERSION` is different from the current version and then act rather than always act and create a PR?

Reply:
This is sorta already handled, I think. Basically when you run `go get x`, if it's the same version that exists, there won't be any changes in the go.mod/sum. If there aren't any changes in the repo, the `peter-evans/create-pull-request` action won't create a PR.